src/pyodide.js

```diff
@@ -120,7 +120,6 @@ var languagePluginLoader = new Promise((resolve, reject) => {
           });
         } else {
           console.error(`Unknown package '${package}'`);
```

Comment:
I imagine this will not fully work if we attempt to a list of remote URLs as input (or a mixture or packages and URLs)? 

If it is the case maybe we can merge this and leave the parent issue open after updating the status? or some decide on some other way of handling this.

Reply:
@rth: Why do you say that?

Reply:
Sorry I misread it for `throw <error>` which would have interrupted the while loop. Nevermind.

Reply:
No problem.  Thanks for the review.